routers/api/v1/repo/issue.go

```diff
@@ -173,6 +173,10 @@ func CreateIssue(ctx *context.APIContext, form api.CreateIssueOption) {
 
 	if form.Closed {
 		if err := issue.ChangeStatus(ctx.User, ctx.Repo.Repository, true); err != nil {
+			if models.IsErrDependenciesLeft(err) {
+				ctx.Error(409, "", fmt.Sprintf("cannot close this issue because it still has open dependencies"))
```

Comment:
Any of these: https://http.cat/412 https://http.cat/417 https://http.cat/424, are better than "Conflict" https://http.cat/409

Also please use `http.Status[...]` 🙂

Reply:
IMO `412` (Precondition Failed) is the closest one here

Reply:
`420` would be also very close

Reply:
@JonasFranzDEV `420` is `enhance your calm` (Twitter troll status). Don't see how that would be relevant 😂